experimental/packages/otlp-transformer/src/logs/types.ts

```diff
@@ -41,67 +31,10 @@ export interface IExportLogsPartialSuccess {
   errorMessage?: string;
 }
 
-/** Properties of a ResourceLogs. */
-export interface IResourceLogs {
-  /** ResourceLogs resource */
-  resource?: IResource;
-
-  /** ResourceLogs scopeLogs */
-  scopeLogs: IScopeLogs[];
-
-  /** ResourceLogs schemaUrl */
-  schemaUrl?: string;
-}
-
-/** Properties of an ScopeLogs. */
-export interface IScopeLogs {
-  /** IScopeLogs scope */
-  scope?: IInstrumentationScope;
-
-  /** IScopeLogs logRecords */
-  logRecords?: ILogRecord[];
-
-  /** IScopeLogs schemaUrl */
-  schemaUrl?: string | null;
-}
-
-/** Properties of a LogRecord. */
-export interface ILogRecord {
-  /** LogRecord timeUnixNano */
-  timeUnixNano: Fixed64;
-
-  /** LogRecord observedTimeUnixNano */
-  observedTimeUnixNano: Fixed64;
-
-  /** LogRecord severityNumber */
-  severityNumber?: ESeverityNumber;
-
-  /** LogRecord severityText */
-  severityText?: string;
-
-  /** LogRecord body */
-  body?: IAnyValue;
-
-  /** LogRecord attributes */
-  attributes: IKeyValue[];
-
-  /** LogRecord droppedAttributesCount */
-  droppedAttributesCount: number;
-
-  /** LogRecord flags */
-  flags?: number;
-
-  /** LogRecord traceId */
-  traceId?: string | Uint8Array;
-
-  /** LogRecord spanId */
-  spanId?: string | Uint8Array;
-}
-
 /**
  * Numerical value of the severity, normalized to values described in Log Data Model.
  */
-export const enum ESeverityNumber {
+export enum ESeverityNumber {
```

Comment:
I had to  convert a few `const enum` to regular enums in this PR to make the JSON Schema validation work correctly. `otlp-transformer` is the only place in the entire codebase that uses `const enum` (everywhere else uses regular enums), so probably these shouldn't have been `const enum` in the first place anyway

Reply:
Presumably the use of `const enum` was motivated by bullet point number 2 in the spec below

>Values of enum fields MUST be encoded as integer values`

https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/specification.md#json-protobuf-encoding

Fortunately, Typebox handles (non-const) enums properly in this case so this isn't an issue

```ts
enum Foo {
  foo = 1,
}
const box = Type.Enum(Foo);

const val: Static<typeof box> = 1;
const key: Static<typeof box> = 'foo'; // this is correctly an error
``